src/DynamoApplications/StartupUtils.cs

```diff
@@ -93,16 +93,20 @@ public static CommandLineArguments Parse(string[] args)
                 //
                 var verbose = string.Empty;
 
+                var convertFile = false;
+
                 bool showHelp = false;
                 var optionsSet = new OptionSet().Add("o=|O=", "OpenFilePath, Instruct Dynamo to open headless and run a dyn file at this path", o => openfilepath = o)
-                .Add("c=|C=", "CommandFilePath, Instruct Dynamo to open a commandfile and run the commands it contains at this path,"+ 
+                .Add("c=|C=", "CommandFilePath, Instruct Dynamo to open a commandfile and run the commands it contains at this path," +
                 "this option is only supported when run from DynamoSandbox", c => commandFilePath = c)
-                .Add("l=|L=", "Running Dynamo under a different locale setting",l => locale = l)
+                .Add("l=|L=", "Running Dynamo under a different locale setting", l => locale = l)
                 .Add("p=|P=", "PresetFile, Instruct Dynamo to import the presets at this path into the opened .dyn", p => presetFile = p)
                 .Add("s=|S=", "PresetStateID, Instruct Dynamo to set the graph to the specified preset by name," +
                 "this can be set to a statename or 'all', which will evaluate all states in the dyn", s => presetStateid = s)
                 .Add("v=|V=", "Verbose, Instruct Dynamo to output all evalautions it performs to an xml file at this path", v => verbose = v)
-                .Add("h|H|help", "Get some help", h => showHelp = h!=null);
+                .Add("x|X", "When used in combination with the 'O' flag, opens a .dyn file from the specified path and converts it to .json." + 
+                "File will have the .json extension and be located in the same directory as the original file.", x => convertFile = x != null)
+                .Add("h|H|help", "Get some help", h => showHelp = h != null);
```

Comment:
to point new users to who are not familiar with the existence of the CLI, and to provide sample usages.